test/getInputFormat.test.js

```diff
@@ -0,0 +1,7 @@
+const getInputFormat = require("../util/getInputFormat.js");
+
+it("Return the right input format", function () {
+  expect(getInputFormat("foo.jpg")).toBe("jpg");
+  expect(getInputFormat("foo.jpg.png")).toBe("png");
+  expect(getInputFormat("foo.jpg.png", "png")).toBe("jpg");
```

Comment:
Hmm, this test makes me realize I find the signature of `getInputFormat` very difficult to read. This assertion was really surprising to me.
But I understand how it works when combined with the options string. I'll try to think of a way to make this API a little clearer.

Reply:
Did you come up with something? I tried to find another name, but keep on circling around "format" and "extension". They are both the same, but `getInputExtension` isn't better IMO. Or maybe something with "source" and "destination" to ditch the word "input" might do the trick.

Reply:
No, let's keep it like this. 
Maybe in the future I'll find some inspiration.